test/opentypeSpec.js

```diff
@@ -28,14 +28,16 @@ describe('opentype.js', function() {
 
     it('can load a font from URL in Node context', function(done) {
         load('https://opentype.js.org/fonts/Roboto-Black.ttf', null, { isUrl: true }).then((font) => {
-            assert.deepEqual(font.names.fontFamily, {en: 'Roboto Black'});
-            assert.deepEqual(font.names.fontFamily, {en: 'Roboto Black'});
+            assert.deepEqual(font.names.macintosh.fontFamily, {en: 'Roboto Black'});
+            assert.deepEqual(font.names.macintosh.fontFamily, {en: 'Roboto Black'});
```

Comment:
why is this done twice?

Reply:
Because I copy-pasted too much! ;) It now checks for the windows name as well, like the other tests do!